src/router.js

```diff
@@ -0,0 +1,30 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+Vue.use(Router);
+
```

Comment:
You can use different approach. Import component in top of the file and use const in component. It can be better when application will grow and more readable.

example :
```
const MainMenu = () => import(''./views/MainMenu.vue'')
const router = new VueRouter({
  routes: [
    {
      path: '/',
      name: 'MainMenu',
      component: MainMenu
    },
  ]
})
```
It's just second option, You can choose what is better for You.